src/lib/components/Speck.react.js

```diff
@@ -243,7 +218,7 @@ Speck.propTypes = {
         atomScale: PropTypes.number,
         relativeAtomScale: PropTypes.number,
         bondScale: PropTypes.number,
-        rotation: PropTypes.arrayOf(PropTypes.number),
+        rotation: PropTypes.shape({}),
```

Comment:
Is there a particular reason this was changed?

Reply:
Yes the rotation prop is a float32array not an array which caused issues when I tried using newer versions of React which are more more strict about proptypes. Therefore I changed the proptype to shape which is a general object.

Reply:
Good catch! Is this the most specific it can get?

Reply:
I don't believe that there is a specific float32array proptype so yes this is the most specific I can get without defining a custom proptype